modules/vantagecloud-lake/pages/vantagecloud-lake-compute-cluster-airflow.adoc

```diff
@@ -0,0 +1,341 @@
+        compute_profile_name="dbt_transformation_profile",
+        compute_group_name="dbt_transformation_group",
+    )
+
+    # run dbt transformation defined in dbt jaffle_shop project
+    transform_data = BashOperator(
+        task_id="transform_data",
+        bash_command="source $PATH_TO_DBT_VENV/bin/activate && dbt seed --profiles-dir $PATH_TO_DBT_PROJECT --project-dir $PATH_TO_DBT_PROJECT &&  dbt run --profiles-dir $PATH_TO_DBT_PROJECT --project-dir $PATH_TO_DBT_PROJECT -s path:models",
+        env={"PATH_TO_DBT_PROJECT": PATH_TO_DBT_PROJECT, "PATH_TO_DBT_VENV": PATH_TO_DBT_VENV},
+    )
+    # suspend compute cluster
+    compute_cluster_suspend_operation = TeradataComputeClusterSuspendOperator(
+        task_id="compute_cluster_suspend_operation",
+        compute_profile_name="dbt_transformation_profile",
+        compute_group_name="dbt_transformation_group",
+    )
+    # decommission compute cluster
+    compute_cluster_decommission_operation = TeradataComputeClusterDecommissionOperator(
+        task_id="compute_cluster_decommission_operation",
+        compute_profile_name="dbt_transformation_profile",
+        compute_group_name="dbt_transformation_group",
+        delete_compute_group=True,
+    )
+    # remove compute group for user
+    remove_compute_group_from_user = TeradataOperator(
+        task_id="remove_compute_group_from_user",
+        sql=r"""
+        MODIFY USER lake_user
+        AS COMPUTE GROUP = NULL
+        """,
+    )
+
+    (
+        compute_cluster_provision_operation
+        >> assign_compute_group_user
+        >> compute_cluster_resume_operation
+        >> transform_data
+        >> compute_cluster_suspend_operation
+        >> compute_cluster_decommission_operation
+        >> remove_compute_group_from_user
+    )
+----
+
+== Configure Airflow
+
+1. Configure the listed environment variables to activate the test connection button, preventing the loading of sample DAGs and default connections in Airflow UI.
++
+[source, bash]
+  export AIRFLOW__CORE__TEST_CONNECTION=Enabled
+  export AIRFLOW__CORE__LOAD_EXAMPLES=false
+  export AIRFLOW__CORE_LOAD_DEFAULT_CONNECTIONS=false
+
+2. Define the path of jaffle_shop project as an environment variable `dbt_project_home_dir`.
++
+[source, bash]
+export dbt_project_home_dir=../../jaffle_shop
++
+NOTE: Change `/../../` to path of jaffle_shop project path.
+
+3. Define the virtual environment path where dbt-teradata installed in <<Install dbt>> as an environment variable `dbt_venv_dir`.
+[source, bash]
+set dbt_venv_dir=/../../dbt_env/bin/activate
++
+NOTE: Change `/../../` to path where virtual environment path defined.
+
+== Start Airflow web server
+1. Run airflow web server
++
+[source, bash]
+----
+airflow standalone
+----
+5. Access the airflow UI. Visit https://localhost:8080 in the browser and log in with the admin account details shown in the terminal.
+
+== Define Airflow connection to Vantage Cloud Lake
```

Comment:
```suggestion
== Define Apache Airflow connection to Vantage Cloud Lake
```

Reply:
Added a couple of more editions basically regarding Apache Product guidelines, specifically that all titles and subtitles should include the full name of the project.

Reply:
Included Apache at specified files